tests/robot-tests/tests/admin/analyst/expired_invitation.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Library             ../../libs/admin_api.py
+Resource            ../../libs/admin-common.robot
+
+Suite Setup         user opens the browser    %{ADMIN_URL}
+Suite Teardown      user closes the browser
+Test Setup          fail test fast if required
+
+Force Tags          Admin    Local    Dev
+
+
+*** Test Cases ***
```

Comment:
I think we also need a UI test that creates an expired invite and then checks it doesn't appear on the `Manage pre-release user access` - before the associated user has tried to accept the invite.

I'm not sure if there are other pages that list invites that also shouldn't include expired invites.

Reply:
As mentioned elsewhere, Cam was happy for expired invites to remain on lists until it was deleted by the user logging in.